config/locales/en/errors.yml

```diff
@@ -147,3 +147,27 @@ en:
               invalid_year: Enter a valid year
               year_too_late: Date of next hearing is too far in the future
               past_not_allowed: Date of next hearing must be in the future
+        steps/case/ioj_form:
+          attributes:
+            types:
+              invalid: Select a criteria
+            loss_of_liberty_justification:
+              blank: Enter more detail
```

Comment:
I know is a pain but I think this may fail stringent accessibility audits, because if there are more than one text area erroring, all will have the same error message read aloud to a screen reader in the summary.

Perhaps, "Enter justification for [xyz]"? i.e. "Enter justification for loss of liberty justification".

Maybe worth checking with @carolinapizatto

Reply:
Good catch, that sounds sensible